docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -0,0 +1,795 @@
+  {
+    path: '',
+    component: NxWelcomeComponent,
+    pathMatch: 'full',
+  },
+  {
+    path: 'products',
+    loadComponent: () =>
+      import('@myngapp/modules/products').then((m) => m.ProductsComponent),
+  },
+  {
+    path: 'orders',
+    loadComponent: () =>
+      import('@myngapp/modules/orders').then((m) => m.OrdersComponent),
+  },
+];
+```
+
+Similarly, navigating to [http://localhost:4200/orders](http://localhost:4200/orders) should now render the `OrdersComponent`.
+
+A couple of notes:
+
+- both, the `ProductsComponent` and `OrdersComponent` are lazy loaded
+- you could go even further and configure routes within the libraries and only import and attach those routes to the application routing mechanism.
+
+## Visualizing your Project Structure
+
+Nx automatically detects the dependencies between the various parts of your workspace and builds a [project graph](/core-features/explore-graph). This graph is used by Nx to perform various optimizations such as determining the correct order of execution when running tasks like `nx build`, identifying [affected projects](/core-features/run-tasks#run-tasks-affected-by-a-pr) and more. Interestingly you can also visualize it.
+
+Just run:
+
+```shell
+nx graph
+```
+
+You should be able to see something similar to the following in your browser (hint: click the "Show all projects" button).
+
+{% graph height="450px" %}
+
+```json
+{
+  "projects": [
+    {
+      "name": "myngapp",
+      "type": "app",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "e2e",
+      "type": "e2e",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "modules-shared-ui",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "modules-orders",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+
+    {
+      "name": "modules-products",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    }
+  ],
+  "dependencies": {
+    "myngapp": [
+      { "source": "myngapp", "target": "modules-orders", "type": "dynamic" },
+      { "source": "myngapp", "target": "modules-products", "type": "dynamic" }
+    ],
+    "e2e": [{ "source": "e2e", "target": "myngapp", "type": "implicit" }],
+    "modules-shared-ui": [],
+    "modules-orders": [],
+    "modules-products": []
+  },
+  "workspaceLayout": { "appsDir": "", "libsDir": "" },
+  "affectedProjectIds": [],
+  "focus": null,
+  "groupByFolder": false
+}
+```
+
+{% /graph %}
+
+Notice how `modules-shared-ui` is not yet connected to anything because we didn't import it in any of our projects. Also the arrows to `modules-orders` and `modules-products` are dashed because we're using lazy imports.
+
+Exercise for you: change the codebase such that `modules-shared-ui` would be connected from `modules-orders` and `modules-products`. Note: you need to restart the `nx graph` command to refresh or run it with the `--watch` flag.
```

Comment:
```suggestion
Exercise for you: change the codebase so that `modules-shared-ui` is used by `modules-orders` and `modules-products`. Note: you need to restart the `nx graph` command to update the graph or run the CLI command with the `--watch` flag.
```